src/utilities/alfalfa/AlfalfaPoint.hpp

```diff
@@ -0,0 +1,111 @@
+/***********************************************************************************************************************
+*  OpenStudio(R), Copyright (c) Alliance for Sustainable Energy, LLC.
+*  See also https://openstudio.net/license
+***********************************************************************************************************************/
+
+#ifndef ALFALFA_POINT_HPP
+#define ALFALFA_POINT_HPP
+
+#include "AlfalfaComponent.hpp"
+#include "utilities/core/Optional.hpp"
+#include "utilities/core/Logger.hpp"
+
+#include "../UtilitiesAPI.hpp"
+
+#include <string>
+#include <json/json.h>
+
+namespace openstudio {
+namespace alfalfa {
+  namespace detail {
+    class AlfalfaPoint_Impl;
+  }
+  class UTILITIES_API AlfalfaPoint
+  {
+   public:
+    AlfalfaPoint();
+
+    AlfalfaPoint(const std::string& display_name);
```

Comment:
I'm noticing this is the only class that has is under the pImpl idiom in that folder. I'm not sure what an Alfalfa point is and why a pImpl makes sense yet.